packages/jaeger-ui-components/src/scroll-page.test.ts

```diff
@@ -140,19 +144,19 @@ describe('scroll-by', () => {
       // cause a `Tween` to be created to get a reference to _onTweenUpdate
       scrollTo(10);
       const { onUpdate } = tweenInstances[0];
-      onUpdate({ value, done: false });
-      expect(window.scrollTo.mock.calls.length).toBe(1);
-      expect(window.scrollTo.mock.calls[0][1]).toBe(value);
+      onUpdate?.({ value, done: false });
+      expect(jest.mocked(window.scrollTo).mock.calls.length).toBe(1);
+      expect(jest.mocked(window.scrollTo).mock.calls[0][1]).toBe(value);
     });
 
     it('discards the in-progress scroll if the scroll is done', () => {
       // cause a `Tween` to be created to get a reference to _onTweenUpdate
       scrollTo(10);
       const { onUpdate, cancel: twCancel } = tweenInstances[0];
-      onUpdate({ value: 123, done: true });
+      onUpdate?.({ value: 123, done: true });
```

Comment:
Why the ?

Reply:
`onUpdate` can be `null` or `undefined`: https://github.com/grafana/grafana/pull/59617/files#diff-4afbe7ba43ccfac4cd8267a282c50a204ca048d775172b79d4e52993b6cc5214R37

Without a `?`, there will be a `Cannot invoke an object which is possibly 'null' or 'undefined'` error.